src/wui/wui.c

```diff
@@ -54,9 +53,9 @@ void StartWebServerTask(void const *argument) {
         if (wui.wui_marlin_vars) {
             marlin_client_loop();
         }
-        osMutexWait(wui_web_mutex_id, osWaitForever);
+        osMutexWait(wui_thread_mutex_id, osWaitForever);
         webserver_marlin_vars = *(wui.wui_marlin_vars);
```

Comment:
here i wanted only to copy used variables, we need a new struct with parameters used